v3/newrelic/tracing.go

```diff
@@ -17,6 +17,10 @@ import (
 	"github.com/newrelic/go-agent/v3/internal/logger"
 )
 
+const (
+	MaxSpanAttributeDBStatementSize = 4096 // int representing max size in bytes
```

Comment:
I think we want to relocate this to `limits.go`- that's where the other limit values (like `attributeValueLengthLimit` already live